### DISCUSSION TITLE
Directory Connector Sync not re-syncing deleted invitee

### DISCUSSION THREAD

**Discussion by compumatter** [dani-garcia/vaultwarden#4149] [General]
Hello,

I have installed the directory connector and the initial GUI test and then Sync goes very well.  A list of users is "invited" as expected.

We are in the testing phase to determine if this product will meet our needs.  As part of the testing I have purposely deleted one of the invited members of the organization who was sync'd and invited after the install.  I then resync'd to see if they would reappear.  They did not.  

So that brings up a few questions.  Since in my inital setup I chose "Overwrite existing organization users based on current sync settings." How do I now get this deleted "invited" user back into being recognized by Bitwarden as being part of the Active Directory sync.  I know I can add the user manually but I have concerns that this manually added user will not be recognized in the same way.

Are there any thoughts on this?

Thanks - Jay

**Comment by BlackDex** [comment]
You need to clear the cache of the directory connector.
The connector doesn't see any changes from the source, save thus isn't going to call the Vaultwarden endpoint.

**Reply by compumatter** [reply]
Thank you!  That was the solve :-) Very grateful for the quick answer on this as well.

**Comment by compumatter** [comment]
I have a follow up question on this if you don't mind.   Is there a down side or up side to me running bwdc clear-cache each time just prior to running my bwdc sync ?

Thank you - Jay

**Reply by BlackDex** [reply]
The only one right now is #4114
I have fixed it already, but not yet merged.

**Comment by compumatter** [comment]
Interesting you brought that one up.  I experienced that today.  Once I implemented your earlier suggestion for clearing cache to get the deleted invited user to reappear - all other members that were previously in the 'invited' state along with the one user that cache clearing restored were all converted to full membership and their invited links disappeared.  It was of course believing that they must create an account to become a full member so that was on todo list to figure out.

I have tested and verified it does still let the user create an account using that same email address that is stated to be a full member so that part still works.  Look forward to the merge.
